lib/jnpr/junos/utils/sw.py

```diff
@@ -986,7 +986,7 @@ def reboot(self, in_min=0, at=None, all_re=True, on_node=None):
                     cmd.append(E('both-routing-engines'))
                 elif self._mixed_VC is True:
                     cmd.append(E('all-members'))
-        if in_min >= 0 and at is None:
+        if in_min >= 0:
```

Comment:
The default value of `in_min` is 0. So this condition is always true.

The test case is failing due to this.

A better logic would be 
```
# in_min given
if in_min > 0:
    append in

# in_min not given but at given
elif at is not None:
    append at

# default case
else:
    apppend in = 0
```

Reply:
It's good idea!